Fastlane/deployment_lanes.rb

```diff
@@ -159,10 +159,6 @@
 
     branch_name = is_hotfix ? "hotfix/#{latest_release_tag}" : "release/#{latest_release_tag}"
 
-    # This is a temporary measure to be able to merge our release.
-    # It will be revised in https://wetransfer.atlassian.net/browse/TMOB-2510
-    sh "git fetch --unshallow"
```

Comment:
How does it behave when releasing the app?

Reply:
It results in an error `[14:51:07]: Exit status of command 'git fetch --unshallow' was 128 instead of 0.`. See the screenshot in the PR description.

Since that was quite obvious, I guess you want to question something else?

Reply:
Is this the same for the iOS app? Sorry, I meant the iOS one, not the macOS.

Reply:
For context: this `--unshallow` was introduced because of issues releasing the iOS app.